pkg/resources/common/validation_test.go

```diff
@@ -60,26 +67,116 @@ func TestValidateRules(t *testing.T) {
 		},
 	}
 
-	for _, testcase := range tests {
-		t.Run("global/"+testcase.name, func(t *testing.T) {
-			err := ValidateRules([]v1.PolicyRule{
-				testcase.data,
+	for _, test := range tests {
+		test := test
```

Comment:
From Go v1.22 this shouldn't be required anymore, but I'm fine keeping it.

Ref: https://go.dev/blog/loopvar-preview